types/cqrs-domain/index.d.ts

```diff
@@ -0,0 +1,707 @@
+// Type definitions for cqrs-domain 2.10
+// Project: https://github.com/adrai/node-cqrs-domain
+// Definitions by: Steven Weiß <https://github.com/blissi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "cqrs-domain" {
```

Comment:
you do not need this line, just remove `declare module "cqrs-domain" {` and the matching closing `}` at the end, and the file should have the same meaning.
this is what is causing the linter to fail.